py-polars/polars/internals/lazy_frame.py

```diff
@@ -1121,10 +1105,7 @@ def agg(self, aggs: Union[tp.List["pli.Expr"], "pli.Expr"]) -> "LazyFrame":
         ...     pl.scan_csv("data.csv")
         ...     .groupby("groups")
         ...     .agg(
-        ...         [
```

Comment:
I think the new lines per expression are much more readable. Could we get this with blackdock?